testing/base/src/main/resources/logback-test.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" debug="true" ?>
```

Comment:
unrelated?

Reply:
well, it's one commit too early but it doesn't hurt